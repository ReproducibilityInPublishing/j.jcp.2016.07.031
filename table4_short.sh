echo "AIMGM" > table4_short.csv

./example2 -l 2 -N 15000 --alpha 0.01 2>/dev/null >> table4_short.csv
./example2 -l 3 -N 15000 --alpha 0.01 2>/dev/null >> table4_short.csv

echo "BFSMGM" >> table4_short.csv

./example2_BFSMGM -l 2 -N 15000 --alpha 0.01 2>/dev/null >> table4_short.csv
./example2_BFSMGM -l 3 -N 15000 --alpha 0.01 2>/dev/null >> table4_short.csv
