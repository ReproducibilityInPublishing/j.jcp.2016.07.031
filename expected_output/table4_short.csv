AIMGM
-1,-1,1,1,0.5,5e-09,1e-08,4,0.01,15000,1,0.210169,5.730608e-04
-1,-1,1,1,0.5,5e-09,1e-08,8,0.01,15000,3,0.514704,1.143205e-04
BFSMGM
-1,-1,1,1,0.5,5e-09,1e-08,4,0.01,15000,1,1.24806,5.730608e-04
-1,-1,1,1,0.5,5e-09,1e-08,8,0.01,15000,3,7.74932,1.143206e-04
